packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -193,9 +186,9 @@ pub trait Store {
     /// [`BaseUri`]: crate::ontology::types::uri::BaseUri
     async fn create_data_type(
         &mut self,
-        data_type: DataType,
+        data_type: &DataType,
         created_by: AccountId,
-    ) -> Result<Persisted<DataType>, InsertionError>;
+    ) -> Result<(), InsertionError>;
```

Comment:
Are we going to make it so that create, update and delete operations all return nothing on success? I think that's reasonable :+1: